pandas/tests/indexes/datetimes/test_partial_slicing.py

```diff
@@ -349,7 +349,7 @@ def test_loc_datetime_length_one(self):
 
     @pytest.mark.parametrize('datetimelike', [
         Timestamp('20130101'), datetime(2013, 1, 1),
-        date(2013, 1, 1), np.datetime64('2013-01-01T00:00', 'ns')])
```

Comment:
Remind me, why was this test removed?

Reply:
This treats `DatetimeIndex` comparison behavior as canonical, which in turn (mostly) follows `Timestamp` behavior.  Neither DTI nor Timestamp allow `date` comparisons.

Reply:
can you add a separate test for date (I know its wrong, but assert that it is wrong)